File: Sem 3-4-5-6 rep/Py/Data Structures in Python/stack.py
stack = []

def isEmpty():
    if len(stack) == 0:
        return True
    else:
        return False

def push(n):
    stack.append(n)

def pop():
    if isEmpty():
        print("Stack is empty.")
    else:
        stack.pop()

def display():
    if isEmpty():
        print ("Stack is empty")
    else:
        print (stack)

while True:
    choice = int (input("Make a choice: "))
    if choice == 1:
        num = int(input("Enter an element to be input in the stack: "))
        push(num)
    elif choice == 2:
        pop()
    elif choice == 3:
        display()

File: Sem 3-4-5-6 rep/C++ VSCode/stack.cpp
#include <iostream>
using namespace std;

int stack[100];
int top = -1;

int push(int val){
    top++;
    stack[top] = val;
}

int pop(){
    if (top == -1){
        cout << "Underflow";
    }
    else {
        top--;        
    }
}

int display(){
    for (int i = top; i>=0; i--){
        cout << stack[i] << endl;
    }
}

int main(){
    int insert, choice;
    do{
        cout << "Make a choice: ";
        cin >> choice;
        switch(choice){
            case 1: cout << "Enter an element: "; cin >> insert; push(insert); break;
            case 2: pop(); break;
            case 3: display(); break;
        }
    }while (choice <= 3);
}